doc/nrf/known_issues.rst

```diff
@@ -2925,21 +3034,29 @@ Samples
 
 .. rst-class:: v2-2-0
 
-NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.
-  When building a sample with support for a front-end module different from nRF21540, the sample uses a non-existing configuration to initialize Tx power data.
+NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module
+  When building a sample with support for a front-end module different from nRF21540, the sample uses a non-existing configuration to initialize TX power data.
   This causes a compilation error because the source file containing code for a generic front-end module is not included in the build.
 
   **Workaround:** Do not use the :kconfig:option:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC` Kconfig option and replace ``CONFIG_GENERIC_FEM`` with ``CONFIG_MPSL_FEM_SIMPLE_GPIO`` in the :file:`CMakeLists.txt` file of the sample.
 
 .. rst-class:: v2-3-0
 
-NCSDK-19858: :ref:`at_monitor_readme` library and :ref:`nrf_cloud_mqtt_multi_service` sample heap overrun.
-  Occasionally, the :ref:`at_monitor_readme` library heap becomes overrun, presumably due to one of the registerred AT event listeners becoming stalled.
+NCSDK-19858: :ref:`at_monitor_readme` library and :ref:`nrf_cloud_mqtt_multi_service` sample heap overrun
+  Occasionally, the :ref:`at_monitor_readme` library heap becomes overrun, presumably due to one of the registered AT event listeners becoming stalled.
   This has only been observed with the :ref:`nrf_cloud_mqtt_multi_service` sample.
 
 Zephyr
 ******
 
+.. rst-class:: v2-3-0 v2-2-0
+
+NCSDK-18426: Fault when using RTT on nRF52 Series devices in specific circumstances
+  Firmware can fault when RTT is enabled on some projects.
+  The underlying cause of this issue has not yet been determined.
+
+  **Workaround:** Disable RTT.
```

Comment:
This can be removed now, the cause of this issue has been identified and the issue is NCSDK-20104 (details in above comment)

Reply:
And I see NCSDK-20104 is fixed for 2.3.0?

Reply:
Yes